app/lib/theme/widgets/FileWidget.tsx

```diff
@@ -77,6 +83,7 @@ const FileWidget: React.FC<WidgetProps> = ({
   }, [fileList, onChange]);
 
   const handleChange = async (e: React.ChangeEvent<HTMLInputElement>) => {
+    setError('');
```

Comment:
@VB-at-Bis I am resetting the error here at the beginning of the `handleChange` component. The reasoning for having it here was that if it was a big file/slow upload the error would be reset at the beginning of each try.